<?xml version="1.0"?>
<launch>
  <arg name="display" default="true"/>

  <arg name="sensor"        default="stereo_pair"/>
  <arg name="tf_prefix"     default="$(arg sensor)"/>

  <arg name="service"       default="false"/>
  
  <arg name="markers"       default="markers"/>

  <arg name="image_left"    default="left/image_rect_color"/>
  <arg name="image_right"   default="right/image_rect_color"/>
  <arg name="stereo_info"   default="stereo_info"/>
  <arg name="dictionary"    default="3"/>

  <include file="$(find aruco_detector)/launch/base_marker.launch" pass_all_args="true">
    <arg name="marker_type" value="aruco"/>
    <arg name="sensor_type" value="stereo"/>
  </include>
</launch>
